OlxProject/pages/product_component_conteiner.rb

```diff
@@ -0,0 +1,8 @@
+require_relative '../tools/all_requires'
+
+class ProductComponentContainer
+  def get_all_products
+     $driver.find_element(id: 'offers_table')
+  end
+
+end
```

Comment:
Will there be any more methods/components in this class? Can it be moved to another class file and be declared there?

Reply:
ye, I`ll complete this class soon.